azext_iot/_params.py

```diff
@@ -232,6 +232,13 @@ def load_arguments(self, _):
             arg_type=get_three_state_flag(),
             help="Flag indicating edge enablement.",
         )
+        context.argument(
+            "connection_string",
+            options_list=["--connection-string", "--cs"],
+            help="Target connection string. This bypasses the IoT Hub registry and generates the SAS token directly"
```

Comment:
It would be helpful to indicate in the description which connection string types are supported.